src/main/java/org/opentripplanner/routing/vehicle_rental/VehicleRentalStationService.java

```diff
@@ -1,35 +1,37 @@
-package org.opentripplanner.routing.bike_rental;
+package org.opentripplanner.routing.vehicle_rental;
```

Comment:
Strictly using a `_` in the package name is not recommended, but I think it is much more readable than not. We should probably make an exception for this in the coding guideline.  I am in favor of keeping the ´_´ here.